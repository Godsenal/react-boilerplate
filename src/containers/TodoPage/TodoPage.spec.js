import React from 'react';
import renderer from 'react-test-renderer';

import { TodoPage } from '.';

const defaultProps = {
  addTodo: () => {},
  deleteTodo: () => {},
  toggleTodo: () => {},
  fetchTodo: () => {},
  setFilter: () => {},
  filteredTodos: [],
  filter: '',
  fetchStatus: '',
};

describe('TodoPage snapshot test', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<TodoPage {...defaultProps} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
