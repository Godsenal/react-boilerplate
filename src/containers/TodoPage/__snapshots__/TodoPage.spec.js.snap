// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoPage snapshot test renders correctly 1`] = `
<div>
  <button
    onClick={[Function]}
    type="button"
  >
    Fetch fake todos STATUS: 
  </button>
  <div>
    <input
      onChange={[Function]}
      onKeyPress={[Function]}
      placeholder="What needs to be done?"
      value=""
    />
  </div>
  <div>
    <span
      className="undefined false"
      onClick={[Function]}
    >
      ALL
    </span>
    <span
      className="undefined false"
      onClick={[Function]}
    >
      INCOMPLETED
    </span>
    <span
      className="undefined false"
      onClick={[Function]}
    >
      COMPLETED
    </span>
  </div>
</div>
`;
